covid/tests.py

```diff
@@ -34,10 +36,12 @@ def test_build(self):
         )
         pmodel = model.build()
         assert isinstance(pmodel, pymc3.Model)
-        # TODO: enable after PR #1 was merged:
-        # assert "date" in pmodel.coords
-        # TODO: make more asserts about dates & coords being part of the model
-        # TODO: assert presence of key random variables
+        # important coordinates
+        assert "date" in pmodel.coords
+        assert "nonzero_date" in pmodel.coords
+        # important random variables
+        for varname in ['r_t', 'seed', 'infections', 'test_adjusted_positive', 'exposure', 'positive', 'alpha']:
```

Comment:
Nit. This test will fail on the first assert. If possible would accumulate all the failures in a list or something and raise them all at once. Not a huge deal though

Reply:
done